cmd/gopogh-server/main.go

```diff
@@ -35,6 +35,12 @@ func main() {
 	// Create an HTTP server and register the handlers
 	http.HandleFunc("/db", db.PrintEnvironmentTestsAndTestCases)
 
+	http.HandleFunc("/env", db.ServeEnvCharts)
```

Comment:
ideally the db Package only talks to DB (insert/or return things) and then main package or "handler" pkg would take care of doing HTTP writing or browser interactions